### DISCUSSION TITLE
How to access Github student discount when institutional mail has been discontinued?

### DISCUSSION THREAD

**Discussion by CodEZ47** [community/community#17718] [GitHub Education]
I was trying to access the Github student developer pack using my institutional email. However, my school, which used to have institutional emails for every student has discontinued the service therefore rendering me incapable of being able to verify my institutional email address to use for discounts. And since my school is listed on the school listings Github requires I add an institutional email address… the institutional email address I can’t verify. What can I do about this inconvenience?
![github](https://user-images.githubusercontent.com/88854432/169645354-dcf750ab-09b9-45f1-9fe3-d09cb328190b.png)

**Comment by Suman373** [comment]
Try contacting your school administration or use a personal mail!

**Comment by mlarasusan** [comment]
Hello [CodEZ47](https://github.com/CodEZ47). If you select a non-academic email or an untrusted academic email, you will be prompted to also upload additional official, dated proof of current school enrollment; such as your student ID, a current registration document, a letter on school letterhead confirming you are a currently-registered student, etc. For more information about the application process and what documents we accept, please read our [blog post](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)
